packages/manager/modules/log-to-customer/src/live-tail/component.js

```diff
@@ -5,9 +5,11 @@ export default {
   controller,
   template,
   transclude: {
-    serviceDescription: '?serviceDescription',
+    tileDescription: '?tileDescription',
+    mainDescription: '?mainDescription',
   },
   bindings: {
+    logsDescription: '<',
```

Comment:
It's just text, right ?
```suggestion
    logsDescription: '@',
```